### PR TITLE
Bump to RxJava 1.1.6

### PR DESCRIPTION
**Pull request by zsxwing**

### CODE REVIEW

File: src/main/scala/rx/lang/scala/Observable.scala [hunk 1]
```diff
@@ -1362,6 +1362,26 @@ trait Observable[+T]
   }
 
   /**
+   * $experimental Returns an [[Observable]] that Requests n` initially from the upstream and then 75% of `n` subsequently after 75% of `n` values have
```
Comment (samuelgruetter) [inline src/main/scala/rx/lang/scala/Observable.scala]: "Requests" lowercase
"n`" needs an opening backtick

File: examples/src/test/scala/examples/RxScalaDemo.scala [hunk 2]
```diff
@@ -1896,4 +1896,23 @@ class RxScalaDemo extends JUnitSuite {
...
+        override def onNext(value: Int): Unit = {
+          println(s"Receive $value")
+          request(1)
+        }
+
+        override def onError(error: Throwable): Unit = error.printStackTrace()
+
+        override def onCompleted(): Unit = println("Done")
+      })
+  }
```
Comment (samuelgruetter) [inline examples/src/test/scala/examples/RxScalaDemo.scala:1917]: nice example :+1:

**Comment by zsxwing** [comment]
updated

**Comment by samuelgruetter** [comment]
LGTM

**Pull request merged**